pysrc/riffq/connection.py

```
@@ -67,30 +67,34 @@ def set_tls(self, crt, key):
+    def get_connection(self, connection_id) -> BaseConnection:
+        conn = self.connections.get(connection_id, None)
         if not conn:
-            conn = self.connection_cls(conn_id, self.executor)
-            self.connections[conn_id] = conn
+            conn = self.connection_cls(connection_id, self.executor)
+            self.connections[connection_id] = conn
         return conn
 
-    def handle_auth(self, conn_id, user, password, host, database=None, callback=callable):
-        conn = self.get_connection(conn_id=conn_id)
+    def handle_auth(self, connection_id, user, password, host, database=None, callback=callable):
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)